packages/core/src/App/Components/Layout/Header/Components/Preloader/header-items.jsx

```diff
@@ -0,0 +1,27 @@
+import ContentLoader from 'react-content-loader';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const HeaderItemsLoader = ({ is_logged_in, speed }) => (
+    <ContentLoader
+        speed={speed}
+        primaryColor={'var(--general-section-1)'}
+        secondaryColor={'var(--general-hover)'}
+        style={{ width: '150px', height: '48px' }}
+    >
+        {is_logged_in ? <LoggedInPreloader /> : <></>}
+    </ContentLoader>
+);
+
+const LoggedInPreloader = () => (
+    <>
```

Comment:
Can we use `React.Fragment` here?

Reply:
`<> </>` is just a short syntax for `React.Fragment`. You can refer to this https://reactjs.org/docs/fragments.html

Reply:
`<> </>` is just a short syntax for `React.Fragment`. You can refer to this https://reactjs.org/docs/fragments.html

Reply:
Yes, I know. My suggestion is to use `React.Fragment` for consistency because we use it for other files in this folder.